content/en/docs/contribute/style/style-guide.md

```diff
@@ -77,36 +79,40 @@ represents.
 
 ### Use bold for user interface elements
 
-<table>
-  <tr><th>Do</th><th>Don't</th></tr>
-  <tr><td>Click <b>Fork</b>.</td><td>Click "Fork".</td></tr>
-  <tr><td>Select <b>Other</b>.</td><td>Select 'Other'.</td></tr>
-</table>
+{{< table caption = "Do and Don't - Bold interface elements" >}}
+Do | Don't
+:--| :-----
+Click <b>Fork</b>. | Click "Fork".
+Select <b>Other</b>. | Select 'Other'. 
+{{< /table >}}
 
 ### Use italics to define or introduce new terms
-
-<table>
-  <tr><th>Do</th><th>Don't</th></tr>
-  <tr><td>A <i>cluster</i> is a set of nodes ...</td><td>A "cluster" is a set of nodes ...</td></tr>
-  <tr><td>These components form the <i>control plane.</i></td><td>These components form the <b>control plane.</b></td></tr>
-</table>
+ 
+{{< table caption = "Do and Don't - Use italics for new terms" >}}
+Do | Don't
+:--| :-----
+A <i>cluster</i> is a set of nodes ... | A "cluster" is a set of nodes ...
+These components form the <i>control plane.</i> | These components form the <b>control plane.</b>
+{{< /table >}}
 
 ### Use code style for filenames, directories, and paths
 
-<table>
-  <tr><th>Do</th><th>Don't</th></tr>
-  <tr><td>Open the <code>envars.yaml</code> file.</td><td>Open the envars.yaml file.</td></tr>
-  <tr><td>Go to the <code>/docs/tutorials</code> directory.</td><td>Go to the /docs/tutorials directory.</td></tr>
-  <tr><td>Open the <code>/_data/concepts.yaml</code><!--to-unbreak-atom-highlighting_--> file.</td><td>Open the /_data/concepts.yaml<!--to-unbreak-atom-highlighting_--> file.</td></tr>
-</table>
+{{< table caption = "Do and Don't - Use code style for filenames, directories, and paths" >}}
+Do | Don't
+:--| :-----
+Open the <code>envars.yaml</code> file. | Open the envars.yaml file.
```

Comment:
```suggestion
Open the <code>envars.yaml</code> file. | Open the `envars.yaml` file.
```
(and elsewhere)

Reply:
@sftim Should I replace the one in "Do" column or "Don't" ?

Reply:
I have replaced the one in column "Do". Let me know if that is not the intended change.

Reply:
Thank you!